EIPS/eip-1078.md

```diff
@@ -7,7 +7,7 @@ status: Draft
 type: Standards Track
 category: ERC
 created: 2018-05-04
-requires: 1077, 725, 681, 191
+requires: 191, 681, 725, 1077
```

Comment:
Does the new validator require they be in order?  I *feel* like it shouldn't as I would *personally* draft an EIP such that the order here is in "relevance to this EIP", so something that is required but not highly relevant would be at the end while something that is required and highly related/relevant would be at the front.

Reply:
Yes it does require the EIPs to be in ascending order. I don't think there should be room for subjective opinions in the preamble, as it is strictly metadata. The body of the EIP should clearly state each required EIP's relevance. This is also standard practice in RFCs: https://tools.ietf.org/html/rfc6247.